docker/Makefile

```
@@ -1,7 +1,7 @@
 all:
 
 # Generate keys for S3 plugin
-rsa-keys:
+rsa_keys:
```

Probably should depend
```suggestion
rsa_keys: private.pem public.pem
```